packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -0,0 +1,260 @@
+import { HTTP_VERBS, _CONVERSATION_URL_, WBXAPPAPI_SERVICE } from '../constants';
```

Comment:
why is this showing up as a new file instead of a renamed one? we're going to lose the history because of that

Reply:
I don't know why git did that but I tried using `git mv` command as well but it still shows deleted and added in the diff. For others it works fine. If you have any suggestion to fix this, please let me know.

Reply:
maybe you have a lot of white space change or different line endings in the renamed file and that's causing the difference between the old and new file to be very big and hence git decides that it's a new file? can you check this and do a binary comparison between the js and ts file and see how many bytes are actually different?

Reply:
Done. Now showing renamed.

Reply:
there are 2 more files in this PR that show as "new" instead of "renamed":
captcha-error.js
logger-proxy.js

Reply:
Done